### PR TITLE
adding default path to xpanse id set

### PR DESCRIPTION
**Pull request by daryakoval**

### CODE REVIEW

File: demisto_sdk/commands/create_id_set/create_id_set.py [hunk 1]
```diff
@@ -85,8 +86,12 @@ def create_command_to_implemented_integration_map(self):
 
     def save_id_set(self):
         if not self.output:
-            self.output = MP_V2_ID_SET_PATH if self.marketplace == MarketplaceVersions.MarketplaceV2.value \
-                else DEFAULT_ID_SET_PATH
+            if self.marketplace == MarketplaceVersions.MarketplaceV2:
+                self.output = MP_V2_ID_SET_PATH
+                if self.marketplace == MarketplaceVersions.XPANSE:
```
Comment (dorschw) [inline demisto_sdk/commands/create_id_set/create_id_set.py:91]: wrong indentation?

File: demisto_sdk/commands/create_id_set/create_id_set.py [hunk 2]
```diff
@@ -85,8 +86,12 @@ def create_command_to_implemented_integration_map(self):
...
+                    self.output = XPANSE_ID_SET_PATH
+                else:
+                    self.output = DEFAULT_ID_SET_PATH
```
Comment (dorschw) [inline demisto_sdk/commands/create_id_set/create_id_set.py:94]: I'd replace it with self.output = {MarketplaceVersions.MarketplaceV2: MP_V2_ID_SET_PATH, MarketplaceVersions.XPANSE:XPANSE_ID_SET_PATH}.get(self.marketplace, DEFAULT_ID_SET_PATH)

File: demisto_sdk/commands/create_id_set/create_id_set.py (hunk 1 above)
Comment (daryakoval) [reply demisto_sdk/commands/create_id_set/create_id_set.py:91]: thanks, your reviews are the best

File: demisto_sdk/commands/create_id_set/create_id_set.py (hunk 2 above)
Comment (daryakoval) [reply demisto_sdk/commands/create_id_set/create_id_set.py:94]: It will break all the places we use self.output... mypy will not accept it.

**Review by Shellyber — APPROVED** [review]

**Pull request merged**